de-lib/src/main/java/org/iplantc/de/apps/client/views/tools/ManageToolsToolbar.ui.xml

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE ui:UiBinder SYSTEM "http://dl.google.com/gwt/DTD/xhtml.ent">
+<ui:UiBinder xmlns:ui="urn:ui:com.google.gwt.uibinder"
+             xmlns:toolbar="urn:import:com.sencha.gxt.widget.core.client.toolbar"
+             xmlns:button="urn:import:com.sencha.gxt.widget.core.client.button"
+             xmlns:menu="urn:import:com.sencha.gxt.widget.core.client.menu"
+             xmlns:MyWidgets="urn:import:org.iplantc.de.apps.client.views.tools">
+
+    <ui:with field="appearance"
+             type="org.iplantc.de.apps.client.views.ManageToolsToolbarView.ManageToolsToolbarApperance" />
+
+    <ui:with field="margins0" type="com.sencha.gxt.core.client.util.Margins">
+        <ui:attributes top="0" right="0" bottom="0" left="0" />
+    </ui:with>
+    <ui:with field="boxData"
+             type="com.sencha.gxt.widget.core.client.container.BoxLayoutContainer.BoxLayoutData">
+        <ui:attributes margins="{margins0}" flex="3" />
+    </ui:with>
+    <ui:with field="boxDataNoFlex"
+             type="com.sencha.gxt.widget.core.client.container.BoxLayoutContainer.BoxLayoutData">
+        <ui:attributes margins="{margins0}" />
+    </ui:with>
+
+    <ui:with field="submitToolTip"
+             type="com.sencha.gxt.widget.core.client.tips.ToolTipConfig">
+        <ui:attributes trackMouse="true" title="{appearance.submitForPublicUse}" />
+    </ui:with>
+
+    <ui:with
+            field="icons"
+            type="org.iplantc.de.resources.client.IplantResources" />
+    <ui:with
+            field="strings"
+            type="org.iplantc.de.resources.client.messages.IplantDisplayStrings" />
```

Comment:
It looks like this `strings` field isn't being used, since label text are coming from the `appearance`.